talent/hiring_process.md

```diff
@@ -0,0 +1,78 @@
+
+- CTO and CEO should be added as reviewers (but should not be bottlenecks to continue).
+
+- Anna posts the job description through Lever on the source{d} website.
+
+### Sourcing
+
+- Lead engineers are responsible to identity people to be contacted and other sourcing channels such as niche job boards. 
+
+- At any given moment the Talent Manager is able to drop in and work on finding more developers.
+
+- Developer lists/sourcing channels should continuously be defined and refreshed after communication between the Talent Manager and the leads. 
+
+- The Talent Manager is responsible to work with the Developer Community team in order to gain people through community actions (Twitter/Blog etc.). 
+
+### First Contact
+
+- Any developer should be contacted by the lead engineers or under their name at least (after together defining the outgoing message). In case the Talent Manager is contacting them, it should be done from the lead's email acccount. It's a motivating factor for the developers to be contacted by the lead engineers. 
+
+### Screening
+
+- If the developer is an applicant, they should be accepted/rejected by the lead engineers during the screening phase. 
+
+- If the developer is approached by us from a specific source (known from meetups, having worked together in the past,  etc.), they should move forward to the next step as it means that the leads have already shown interest.
+
+### First Skype/Zoom/Phone Interview
+
+- A quick Skype/Phone/Zoom call should be available to every developer who is interested in the roles*. It is a general discussion to know more about the background and experience of the candidate. And to share as much as possible from our side about the team & company. 
+
+\* There a noteable exceptions to this, such as higher volume junior roles where we directly ask the developer to do the coding challenge before a call. 
+
+- Lead engineers are responsible to make the calls as it's a motivating factor for the developers. Leads will also be able to assess early in the process if the interview process should continue or not. 
+
+### Coding Challenges 
+
+- Every candidate who passes the first Skype interview, should complete the coding challege. 
+
+- In case of the Machine Learning team, there is an additional ML task necessary to be solved.
+
+- Coding Challeges are being sent to the candidates by Anna and the results are being reviewed by the lead engineers. 
+
+### On-site Interview
+ 
+- This is the last step in which it's  decided if the candidate will receive an offer or not. It might last a full day or be split across two half-days, depending on the availability of the candidate. 
+
+- The interviews are the followings: 
+
+* Kickoff with Maximo, QA: to know the candidate better
+* Design interview: to see how the candidate would design a solution to a given technical problem working with a several of our engineers
```

Comment:
In case of a candidate being interviewed for infrastructure we have to make sure that design interview covers what we need. It has happened in the past that I had to make an additional infra interview as design was too wide (candidate was also considered for applications team)

Reply:
This is a very good point! Let's add a reference to this to the guide @tsolakoua